kfdefs/overlays/moc/smaug/ray-bu/manager.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kuberay-operator
+  namespace: ray-bu
+  labels:
+    app.kubernetes.io/name: kuberay
+    app.kubernetes.io/component: kuberay-operator
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kuberay
+      app.kubernetes.io/component: kuberay-operator
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: kuberay
+        app.kubernetes.io/component: kuberay-operator
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      serviceAccountName: kuberay-operator
```

Comment:
Where is this Service account defined? I think you're missing a manifest in the PR.😉